flare/gp.py

```diff
@@ -228,6 +264,13 @@ def en_kern_vec(self, x: AtomicEnvironment) -> np.ndarray:
         return k_v
 
     def set_L_alpha(self):
+        """
+        Invert the covariance matrix, setting L (a lower triangular
+        matrix s.t. L L^T = (K + sig_n^2 I) ) and alpha, an inversion of the
+        vector of training labels.

```

Comment:
Alpha is the inverse of the covariance matrix multiplied by the vector of training labels.

Reply:
Whoops, thanks. Fixed!